python/IECoreArnoldTest/RendererTest.py

```diff
@@ -1851,19 +1851,33 @@ def testOSLShaders( self ) :
 						"spline" : IECore.SplinefColor3f(
 							IECore.CubicBasisf.bSpline(),
 							[
+								( 0, imath.Color3f( 0 ) ),
 								( 0, imath.Color3f( 0.25 ) ),
-								( 0, imath.Color3f( 0.25 ) ),
-								( 1, imath.Color3f( 0.5 ) ),
 								( 1, imath.Color3f( 0.5 ) ),
+								( 1, imath.Color3f( 1 ) ),
```

Comment:
I understand the `floatSplineHandle` stuff added below, where we're testing the duplicate endpoints we add to work around the oddness of OSL discarding linear endpoints. But I don't understand why we're changing this existing test - the original points had duplicated endpoints equivalent to the ones that SplinePlug would generate, but these new points have duplicate _positions_ but with different _values_. I don't think we'd ever generate points like that would we?

Reply:
Conclusion from chat just now : we do want to change this, but we want to make the positions as well as the values unique.

Reply:
Addressed in place